src/mlpack/core/cv/metrics/r2_score.hpp

```diff
@@ -43,11 +43,13 @@ namespace cv {
  * For example, a model having R2Score = 0.85, explains 85 \% variability of
  * the response data around its mean.
  */
+
```

Comment:
```suggestion
 *
 * @tparam AdjustedR2 If true, then the adjusted R2 score will be used.
 *     Otherwise, the regular R2 score is used.
 */
```
Now we can use Doxygen to document the template parameter. :)